bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/form/ReservedProperties.java

```diff
@@ -162,6 +162,7 @@ private ReservedProperties() {
     public static final String FD_ENABLE_AUTO_SAVE = "fd:enableAutoSave";
     public static final String FD_AUTO_SAVE_STRATEGY_TYPE = "fd:autoSaveStrategyType";
     public static final String FD_AUTO_SAVE_INTERVAL = "fd:autoSaveInterval";
+    public static final String FD_AUTOCOMPLETE = "fd:autoComplete";
```

Comment:
Can we rename this to `fd:enableSuggestions` ? The term "autoComplete" may cause confusion with HTML5's autocomplete feature.